src/screens/searchResult/screen/tabs/topics/container/topicsResultsContainer.js

```diff
@@ -0,0 +1,60 @@
+import { useState, useEffect } from 'react';
+import get from 'lodash/get';
+import { withNavigation } from 'react-navigation';
+import { connect } from 'react-redux';
+
+import ROUTES from '../../../../../../constants/routeNames';
+
+import { getTrendingTags } from '../../../../../../providers/hive/dhive';
+import { getLeaderboard } from '../../../../../../providers/ecency/ecency';
+import { isCommunity } from '../../../../../../utils/communityValidation';
+
+const OtherResultContainer = (props) => {
+  const [tags, setTags] = useState([]);
```

Comment:
also by default, fetch 50 tags with `getTrendingTags` and filter out communities from result.